Cargo.toml

```diff
@@ -15,7 +15,7 @@ rust-version = "1.65.0"
 
 [dependencies]
 miniconf_derive = { path = "miniconf_derive" , version = "0.6" }
-serde-json-core = "0.5.0"
+serde-json-core = { git = "https://github.com/quartiq/serde-json-core.git", branch = "rj/ser-de-pub" }
```

Comment:
Can this get updated now? Maybe serde-json-core can cut a release

Reply:
#149 does that.
Let's do the release prep in a second step.